gdk/wizard/WizardChecker.py

```diff
@@ -77,3 +83,70 @@ def is_valid_custom_build_command(self, input_value):
             return False
         except ValueError:
             return len(input_value) > 0
+
+    def check_build_options(self, input_value):
+        try:
+            # Convert the input string to a dictionary (object)
+            input_obj = json.loads(input_value)
+        except json.JSONDecodeError:
+            return False
+
+        if not isinstance(input_obj, dict):
+            return False
+
+        if len(input_obj) > 2:
+            return False
+
+        if "excludes" not in input_obj or "zip_name" not in input_obj:
+            return False
+
+        excludes = input_obj.get("excludes")
+        if (
+            not isinstance(excludes, list)
+            or len(excludes) < 1
+            or not all(isinstance(item, str) for item in excludes)
+        ):
+            return False
+
+        zip_name = input_obj.get("zip_name")
```

Comment:
If `zip_name` does not exist, it is returned as `None`.
Validate only when `zip_name` exists.